packages/ddex/ingester/parser/ern38x.go

```diff
@@ -176,6 +176,24 @@ func processReleaseNode(rNode *xmlquery.Node, soundRecordings *[]SoundRecording,
 	isrc := safeInnerText(rNode.SelectElement("ReleaseId/ISRC"))
 	releaseType := safeInnerText(rNode.SelectElement("ReleaseType"))
 
+	// Release IDs
+	ddexReleaseIDs := &common.ReleaseIDs{
+		PartyID:       safeInnerText(rNode.SelectElement("ReleaseId/PartyId")),
+		CatalogNumber: safeInnerText(rNode.SelectElement("ReleaseId/CatalogNumber")),
```

Comment:
i'm wondering if there would ever be a case where a track transfers rights and we want to know the DPID attribute of the CatalogNumber / ProprietaryId. i'm thinking probably not but just writing this down in case it comes up later. we should be able to use the ddex app's key as a way to identify which distributor the track comes from anyway